src/unlimited.cpp

```diff
@@ -1554,156 +1693,151 @@ bool TestConservativeBlockValidity(CValidationState& state, const CChainParams&
+                            "  }\n"
+                            "\nExamples:\n" +
+                            HelpExampleCli("getstat", "") + HelpExampleRpc("getstat", "") + "\n" + specificIssue);
 
     UniValue ret(UniValue::VARR);
 
     string seriesStr;
     if (params.size() < 2)
-      seriesStr = "total";
-    else seriesStr = params[1].get_str();
-    //uint_t series = 0; 
-    //if (series == "now") series |= 1;
-    //if (series == "all") series = 0xfffffff;
+        seriesStr = "total";
+    else
+        seriesStr = params[1].get_str();
+    // uint_t series = 0;
+    // if (series == "now") series |= 1;
+    // if (series == "all") series = 0xfffffff;
     LOCK(cs_statMap);
 
     CStatBase* base = FindStatistic(params[0].get_str().c_str());
     if (base)
-      {
+    {
         UniValue ustat(UniValue::VOBJ);
         if (seriesStr == "now")
-          {
-	    ustat.push_back(Pair("now", base->GetNow()));
-	  }
+        {
+            ustat.push_back(Pair("now", base->GetNow()));
+        }
         else if (seriesStr == "total")
-          {
-	    ustat.push_back(Pair("total", base->GetTotal()));
-	  }
+        {
+            ustat.push_back(Pair("total", base->GetTotal()));
+        }
         else
-	  {
-            UniValue series = base->GetSeries(seriesStr,count);
-	    ustat.push_back(Pair(seriesStr,series));
-	  }
+        {
+            UniValue series = base->GetSeries(seriesStr, count);
+            ustat.push_back(Pair(seriesStr, series));
+        }
 
-        ret.push_back(ustat);  
-      }
+        ret.push_back(ustat);
+    }
 
     return ret;
 }
 
-static const CRPCCommand commands[] =
-{ //  category              name                      actor (function)         okSafeMode
-  //  --------------------- ------------------------  -----------------------  ----------
+static const CRPCCommand commands[] = {
+    //  category              name                      actor (function)         okSafeMode
+    //  --------------------- ------------------------  -----------------------  ----------
     /* P2P networking */
-    { "network",            "settrafficshaping",      &settrafficshaping,      true  },
-    { "network",            "gettrafficshaping",      &gettrafficshaping,      true  },
-    { "network",            "pushtx",                 &pushtx,                 true  },
-    { "network",            "getexcessiveblock",      &getexcessiveblock,      true  },
-    { "network",            "setexcessiveblock",      &setexcessiveblock,      true  },
-    { "network",            "expedited",              &expedited,              true  },
+    {"network", "settrafficshaping", &settrafficshaping, true},
+    {"network", "gettrafficshaping", &gettrafficshaping, true}, {"network", "pushtx", &pushtx, true},
+    {"network", "getexcessiveblock", &getexcessiveblock, true},
+    {"network", "setexcessiveblock", &setexcessiveblock, true}, {"network", "expedited", &expedited, true},
 
```

Comment:
I think this whole section needs to be restored and it would be useful to make clang not look at this part.